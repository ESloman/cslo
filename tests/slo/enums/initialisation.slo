enum Colours {
    BLACK,
    BLUE,
    YELLOW,
    ORANGE
}

println(Colours);
