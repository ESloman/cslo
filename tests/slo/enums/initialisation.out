enum Colours: {YELLOW: 2, ORANGE: 3, BLACK: 0, BLUE: 1}
